### ISSUE TITLE
Игра не поощрает стрельбу пулями на дальнее расстояние

### ISSUE THREAD

**Issue by Motzord** [TauCetiStation/TauCetiClassic#7130]
#### Подробное описание проблемы
Почему-то закодено, что если пуля улетает за зону видимости стрелка и пролетает >7 тайлов, то ее урон значительно режется (вроде вдове и продолжает резаться за каждые несколько тайлов) итого, если стрелять из коридора мостика в сторону условного брига, в зазевавшегося офицера прилетит конфети вместо куска свинца.
#### Что должно было произойти
Игрок может стрелять в сторону условного противника и сдерживать его.
#### Что произошло на самом деле
Пульки через 14+ тайлов прикольно шлепают по кукле нанося <10 брута.
#### Как повторить
Поставьте куклу в одном конце коридора, начинайте спамить боевыми пульками с другого конца. Удивитесь
#### Дополнительная информация:
![image](https://user-images.githubusercontent.com/45365635/117533798-69199800-aff7-11eb-8f09-3e489e127f19.png)
![image](https://user-images.githubusercontent.com/45365635/117533820-7e8ec200-aff7-11eb-871d-3c398e037d3b.png)

Шесть пулек с пилы, выпущенные на расстоянии 32 тайлов дамажат <10 брута, я считаю настолько все плохо быть не должно

**Comment by Motzord** [comment]
Таким кстати образом страдают офицеры, которые стреляют в спину убегающей ассистухи с глока.

**Comment by Gort133** [comment]
> Таким кстати образом страдают офицеры, которые стреляют в спину убегающей ассистухи с глока.

Пусть страдают

**Comment by Motzord** [comment]
> потому что умирать от того что ты танцуешь в упор с врагом =/= умирать от случайно прилетевшей с другого конца карты шрапнели
> на свою смерть когда видишь врага ты повлиять можешь, а когда в тебя прилетает случайная загогулина - нет. не весело однако

Я не прошу чтобы пулька с 32 тайлов дамажила столько же, сколько изначально, но увеличить радиус опиздюливания хотя бы на 14 тайлов будет хорошо. При перестрелках можно хотя бы будет уйти за экран и стрелять вслепую.

**Comment by AndroBetel** [comment]
ну ниче и не сказал короче почему чтобы больно била с дистанции это хорошо а наоборот это плохо, тупо "я хочу сделайте так", какие-то фантазии про подавление и сдерживание противника, которые и так работают, потому что даже 10 урона это ощутимо, а больше и не надо чтобы в тебя из ниоткуда прилетали префаеры

**Comment by Motzord** [comment]
> ну ниче и не сказал короче почему чтобы больно била с дистанции это хорошо а наоборот это плохо, тупо "я хочу сделайте так", какие-то фантазии про подавление и сдерживание противника, которые и так работают, потому что даже 10 урона это ощутимо, а больше и не надо чтобы в тебя из ниоткуда прилетали префаеры

а ты считаешь нормой, что пуля пролетев 8 тайлов выдает в два раза меньше урона?

**Comment by AndroBetel** [comment]
да, потому что это называется баланс. ломающиеся руконоги из ниоткуда это не весело
